android/expoview/build.gradle

```diff
@@ -113,6 +113,7 @@ configurations.all {
 // WHEN_VERSIONING_REMOVE_FROM_HERE
 // Import gradle helpers for unimodules.
 apply from: "../../node_modules/react-native-unimodules/gradle.groovy"
+apply from: "unimodules.gradle.kts"
```

Comment:
Can we move this `unimodules.gradle.kts` into `gradle.kts` in `react-native-unimodules`?

Reply:
kotlin gradle script seems to require naming in form *.gradle.kts